compiler/rustc_builtin_macros/src/lib.rs

```diff
@@ -77,28 +78,29 @@ pub fn register_builtin_macros(resolver: &mut dyn ResolverExpand) {
         cfg: cfg::expand_cfg,
         column: source_util::expand_column,
         compile_error: compile_error::expand_compile_error,
+        concat: concat::expand_concat,
```

Comment:
Given that you ordered this list alphabetically, can you add `// tidy-alphabetical-start` and `-end` here?

Reply:
Done